### PR TITLE
CMOR fixes for CMIP6 (variable clcalipso)

### PR DESCRIPTION
**Pull request by axel-lauer**
Corrects standard_name of vertical coordinate alt40 used by variable clcalipso (CFMon) from the three CMIP6 models CNRM-CM6-1, CNRM-ESM2-1 and IPSL-CM6A-LR.

### CODE REVIEW

File: esmvalcore/cmor/_fixes/cmip6/cnrm_cm6_1.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class Clcalipso(Fix):
+    """Fixes for clcalipso."""
+
+    def fix_metadata(self, cubes):
+        """
+        Corrects alt40 coordinate standard_name.
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
```
Comment (schlunma) [inline esmvalcore/cmor/_fixes/cmip6/cnrm_cm6_1.py]:
```suggestion
        cubes : iris.cube.CubeList
```

File: esmvalcore/cmor/_fixes/cmip6/cnrm_esm2_1.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+class Clcalipso(Fix):
+    """Fixes for clcalipso."""
+
+    def fix_metadata(self, cubes):
+        """
+        Corrects alt40 coordinate standard_name.
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
```
Comment (schlunma) [inline esmvalcore/cmor/_fixes/cmip6/cnrm_esm2_1.py]:
```suggestion
        cubes : iris.cube.CubeList
```

File: esmvalcore/cmor/_fixes/cmip6/ipsl_cm6a_lr.py [hunk 3]
```diff
@@ -44,3 +44,28 @@ def fix_metadata(self, cubes):
...
+class Clcalipso(Fix):
+    """Fixes for clcalipso."""
+
+    def fix_metadata(self, cubes):
+        """
+        Corrects alt40 coordinate var_name.
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
```
Comment (schlunma) [inline esmvalcore/cmor/_fixes/cmip6/ipsl_cm6a_lr.py]:
```suggestion
        cubes : iris.cube.CubeList
```

**Comment by axel-lauer** [comment]
Added these changes to PR https://github.com/ESMValGroup/ESMValCore/pull/551.

**Pull request closed without merging**